plugins/playlist/src/components/PlaylistEditDialog/PlaylistEditDialog.tsx

```diff
@@ -74,13 +75,28 @@ export const PlaylistEditDialog = ({
 }: PlaylistEditDialogProps) => {
   const classes = useStyles();
   const identityApi = useApi(identityApiRef);
-
+  const playlistApi = useApi(playlistApiRef);
+  const playlistPromise = useRef(playlistApi.getAllPlaylists({}));
+  const [editingOtherFields, setEditingOtherFields] = useState(false);
   const { loading: loadingOwnership, value: ownershipRefs } =
     useAsync(async () => {
       const { ownershipEntityRefs } = await identityApi.getBackstageIdentity();
       return ownershipEntityRefs;
     }, []);
 
+  const nameIsUnique = async (name: string, isEditing: boolean) => {
```

Comment:
I'm still not sure why we need the `isEditing` param?

Reply:
It is for when user tries to edit a playlist and he just updated the description or other field except title field its should not throw the duplicate error

Reply:
Isn't that already satisfied by the `name !== playlist.name` check? If someone is not changing the name then this conditional should fail since the name has not changed right? and it should fall back to the `return true;` line below which indicates there is no error? 

Let me know if I'm missing something

Reply:
> I'm still not sure why we need the `isEditing` param? 

When user tries to edit the playlist and if he start enters some value in the title field the validation will be started or else its should not throw any errors 

If user wants to edit just a description we need to allow them to edit it without any validations.

Will try to remove the isEdit param and test it once and get back to you.